docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,88 @@
+"""
+Coriolis parameters across the world
+====================================
+
+This code computes the Coriolis parameter for an area and stores it in a cube.
+
+It creates and plots a map showing the Coriolis parameters, overlaid on top of
+an Orthographic projection of the globe.
+
+"""
+
+import cartopy.crs as ccrs
+import iris
+from iris.coord_systems import GeogCS
+import iris.plot as iplt
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 200
+    longitudes = np.linspace(-180.0, 180.0, resolution)
+    latitudes = np.linspace(-90.0, 90.0, resolution)
+    lon2d, lat2d = np.meshgrid(longitudes, latitudes)
+
+    # Omega is the Earth's rotation rate, expressed in radians per second
+    omega = 7.29e-5
+
+    # Now to create our data
+    data = 2. * omega * np.sin(np.deg2rad(lat2d))
+
+    # We now need to define a coordinate system for the plot.
+    # Here we'll use GeogCS; 6371229 is the radius of the Earth in metres.
+    cs = GeogCS(6371229)
+
+    # The Iris coords module turns the latitude list into a coordinate array.
+    # Coords then applies an appropriate standard name and unit to it.
+    lat_coord = iris.coords.DimCoord(latitudes,
+                                     standard_name='latitude',
+                                     units='degrees',
+                                     coord_system=cs)
+
+    # The above process is repeated for the longitude coordinates.
+    long_coord = iris.coords.DimCoord(longitudes,
+                                      standard_name='longitude',
+                                      units='degrees',
+                                      coord_system=cs)
+
+    # The next step is to add vertical structure to the cube.
+    # This cube has only one level, but a cube with multiple model levels
+    # can be made by feeding in a list rather than a single value.
+    height_coord = iris.coords.DimCoord(100.,
```

Comment:
Well, think about it like this: we are demonstrating how to create metadata for a quantity we have derived ourselves. There is no logical reason why you would introduce this level coordinate, since the Coriolis parameter is independent of height. Adding the height coordinate teaches users to add spurious metadata to their cubes, which is not desirable. If you can't justify it in the context of the example, it shouldn't be there.

If you can think of another scalar coordinate that makes sense in the context of this example, please go ahead and add it, if not then I suggest you don't bother.